localstack/services/events/events_listener.py

```diff
@@ -138,6 +139,45 @@ def handle_put_rule(data: Dict):
     return True
 
 
+def handle_create_connection(data: Dict, headers: Dict):
+    name = data.get("Name")
+    auth_type = data.get("AuthorizationType")
+    errors = []
+
+    pattern = re.compile("^[\\.\\-_A-Za-z0-9]+$")
+    if not pattern.match(name):
+        error = (
+            "%s at 'name' failed to satisfy: Member must satisfy regular expression pattern: [\\.\\-_A-Za-z0-9]+"
+            % name
+        )
+        errors.append(error)
+
+    if len(name) >= 65:
```

Comment:
```suggestion
    if len(name) > 64:
```
a bit of a nit but I feel this expresses the fact that it's a complement to `<= 64` a bit better.